### PR TITLE
Remove the mongodb benchmark.

### PR DESCRIPTION
**Pull request by cmccoy**
Deprecated in favor of mongodb_ycsb.

See #294.

### CODE REVIEW

**Comment by ehankland** [comment]
LGTM

File: perfkitbenchmarker/benchmark_sets.py [hunk 1]
```diff
@@ -32,9 +32,9 @@
         BENCHMARK_LIST: [
             'aerospike', 'cassandra_stress', 'object_storage_service',
             'cluster_boot', 'copy_throughput', 'coremark', 'fio',
-            'hadoop_terasort', 'hpcc', 'iperf', 'mesh_network', 'mongodb',
-            'netperf', 'ping', 'redis', 'speccpu2006',
-            'block_storage_workload', 'sysbench_oltp', 'unixbench']
+            'hadoop_terasort', 'hpcc', 'iperf', 'mesh_network', 'mongodb_ycsb',
```
Comment (voellm) [inline perfkitbenchmarker/benchmark_sets.py]: Not that you are adding this to the STANDARD_SET you can remove it from the google_set.  I'd comment there but Github only allows commenting on code changed.

**Comment by voellm** [comment]
LGTM ... though you can remove the mongodb_ycsb from the google_set as commented.

File: perfkitbenchmarker/benchmark_sets.py (hunk 1 above)
Comment (cmccoy) [inline perfkitbenchmarker/benchmark_sets.py]: done.

**Pull request merged**